engine/src/main/java/org/terasology/rendering/dag/nodes/CopyImageToHMDNode.java

```diff
@@ -117,5 +119,14 @@ private void renderFinalStereoImage(RenderingStage renderingStage) {
                 GL11.glFinish();
                 break;
         }
+
+        // Bind the default FBO. The DAG does not recognize that this node has
+        // bound a different FBO, so as far as it is concerned, FBO 0 is still
+        // bound. As a result, without the below line, the image is only copied
+        // to the HMD - not to the screen as we would like. To get around this,
+        // we bind the default FBO here at the end.  This is a bit brittle
+        // because it assumes that FBO 0 is bound before this node is run.
+        // TODO: break this node into two different nodes that use addDesiredStateChange(BindFBO...))
+        new BindFBOTask(0,DEFAULT_FRAME_BUFFER_URN).execute();
```

Comment:
1. you are right it's deprecated, but this is meant to be temporary and to misuse tasks is more deprecated. :smile: 

2. Hmm. Strange. Might be worth to pop a question in an OpenVR forum perhaps? My impression is that part of the code completely overwrites the content of those buffers, so there shouldn't be a need clear them. Meanwhile you might want to leave a `// TODO - review need for clearing` in there? 

3. I feel by the time we get to this part of the code we are squarely in 2D processing mode: the only input data you are using is from the FINAL fbo which only has a color attachment: depth data at this stage is long gone. So, I'm struggling to see how the depth buffer might be involved even for the purpose you mention.